build-index.sh

```diff
@@ -4,7 +4,13 @@
 
 hash realpath 2>/dev/null || function realpath() { cd "$1"; pwd -P; }
 
-master=${GITHUB_MASTER:-master}
+if DEFAULT_BRANCH=$(git rev-parse --abbrev-ref origin/HEAD 2>/dev/null); then
+    DEFAULT_BRANCH="${DEFAULT_BRANCH#*/}"
+else
+    DEFAULT_BRANCH=master
+fi
```

Comment:
I switched to

```bash
DEFAULT_BRANCH=$(git rev-parse --abbrev-ref origin/HEAD 2>/dev/null | sed 's,.*/,,')
if [ "$DEFAULT_BRANCH" = "HEAD" ]; then
    git remote set-head -a origin
    DEFAULT_BRANCH=$(git rev-parse --abbrev-ref origin/HEAD 2>/dev/null | sed 's,.*/,,')
fi
```

because `rev-parse` returns `origin/HEAD` when there's no remote HEAD set, and we can actually fix the problem automatically.